src/components/CreateNewProjectWizard/CreateNewProjectWizard.js

```diff
@@ -103,10 +105,15 @@ class CreateNewProjectWizard extends PureComponent<Props, State> {
   };
 
   finishBuilding = (project: Project) => {
+    const { onboardingStatus } = this.props;
+
+    // Get onboardingStatus to check if sidebar instructions should display
+    const onboardingCompleted = onboardingStatus === 'done';
```

Comment:
I would actually do this check in `mapStateToProps`. I like the idea of giving components as little information as they need, and `CreateNewProjectWizard` doesn't actually need to know the exact status, it just needs to know if it's done or not.

So I'd make the prop itself `isOnboardingCompleted` and then just pass it through this component to the action.

Another nice touch we could do is to add a selector function to `onboarding-status.reducer.js`, like:

```js
export const getOnboardingCompleted = (state: GlobalState) => 
  getOnboardingStatus(state) === 'done';
```

Then in this component you would just do:

```js
const mapStateToProps = state => ({
  isOnboardingCompleted: getOnboardingCompleted(state),
})
```

The reason this is nice is that the `onboarding-status.reducer` "owns" the bit of information about what the "done" state is. Let's say in the future we want to change how onboarding statuses work, we'd only have to change that one file, rather than finding all the places where it checks for raw values in components.

Reply:
Oh man, this is why I love reviews, such valuable info. Never thought to do this in mapStateToProps and in the reducer but it's so much cleaner. Plus the reducer has the matching `getOnboardingStatus` so it makes total sense to keep this done check in there as well.